client/my-sites/upgrades/domain-management/dns/email-provider.jsx

```diff
@@ -75,7 +79,7 @@ class Office365 extends Component {
 						<FormButton
 							disabled={ ! isDataValid || this.state.submitting }
 							onClick={ this.onAddDnsRecords }>
-							{ translate( 'Set up Office 365' ) }
+							{ translate( 'Set up ' ) }{ name }
```

Comment:
Thank you for the example and the links. This was actually one of those other things that wasn't exactly sure about how to handle it when I originally wrote it, but then forgot to go back and try to figure out the right way to do it.